### PR TITLE
Crisis care criteria (WIP)

### LINKED ISSUE #2: Pull data from Ryan's existing sheet
This is the sheet he copy/pastes every day.

https://docs.google.com/spreadsheets/d/1ZhwP0GZTz50myibSaWsMXOVQKx9DQaJO4rN1i58Rrjc/edit#gid=1300555045

We want to pull the ICU and inpatient beds data to start.

### PR DESCRIPTION
**Pull request by patricksheehan**
Closes #2

### CODE REVIEW

File: covid/constants.py [hunk 1]
```diff
@@ -0,0 +1 @@
+PATH_TO_SERVICE_ACCOUNT_KEY = "service-account-key.json"
```
Comment (lucasmbrown) [inline covid/constants.py:1]: Nice!

**Review by lucasmbrown — APPROVED** [review]
LGTM! We might refactor later but we can get this in for now.

File: main.py [hunk 2]
```diff
@@ -21,24 +26,46 @@
...
-    df = extract_covidtracking_historical_data()
-
-    transformed_df = transform_covidtracking_data(df=df)
-
     client, credentials = get_sheets_client(
         credential_file_path=os.path.abspath(PATH_TO_SERVICE_ACCOUNT_KEY)
     )
 
+    cdc_current_df = extract_current_cdc_data()
+    cdc_historical_df = extract_historical_cdc_data(credentials)
```
Comment (patricksheehan) [inline main.py:41]: The flow here is that we pull the new data using Brice's scraper, we pull our "db" data from a google sheet, we mash the two to get all the historical data + the current data on the CDC site, then transform and upload.

Comment (patricksheehan) [reply main.py:41]: @lucasmbrown ^

Comment (patricksheehan) [reply main.py:41]: Each time, the new CDC data is added to our historical data set

File: covid/transform.py [hunk 3]
```diff
@@ -513,14 +533,61 @@ def transform_covidtracking_data(df):
...
 
     # Copy state values into column called "State" instead of "state".
     df[STATE_FIELD] = df[STATE_SOURCE_FIELD]
 
     return df
 
 
+def transform_cdc_data(cdc_current_df, cdc_historical_df):
+    # Add date to index
+    cdc_df = pd.concat([cdc_current_df, cdc_historical_df], axis=0)
```
Comment (lucasmbrown) [inline covid/transform.py:578]: Got it -- so based on this and your very helpful comment, for now, we'll only have data from the first time we ran @BriceWolfgang's scraper (6/10)? 

If I'm understanding that correctly, maybe we can create a backlog issue to convert the CSV we found of bed data and upload it once to this sheet.

**Review by lucasmbrown — APPROVED** [review]
Looking good!

File: covid/transform.py (hunk 3 above)
Comment (patricksheehan) [reply covid/transform.py:578]: Yes that sounds like a plan!

**Pull request merged**